shared/src/main/scala/mlscript/codegen/Scope.scala

```
@@ -200,6 +200,8 @@ class Scope(name: Str, enclosing: Opt[Scope]) {
       declareTrait(name, tparams map { _.name }, body, mthdDefs)
     case TypeDef(Cls, TypeName(name), tparams, baseType, _, members, _) =>
       declareClass(name, tparams map { _.name }, baseType, members)
+    case TypeDef(Nms, TypeName(name), _, body, _, _, _) =>
+      declareTypeAlias(name, Ls(), body) // TODO: deal with namespaces
```

Silent errors are the worst! Better raise a code-gen exception.